### DISCUSSION TITLE
販賣機可以賣相親個資！

### DISCUSSION THREAD

**Discussion by benestia** [1101-GS4719-C/Python-Community-GS4719-C#1123] [Ideas]
不得不說想到販賣機賣相親個人資料真的很聰明。因為相親如果是以人作為仲介的話，可能會有人覺得不好意思，像是擔心自己年紀太大、身材不夠好，就索性不找了。但如果是透過無人商店賣的話就不會有這種問題了。
但我在想如果罐罐被買走，但對方不喜歡，沒被選到的話要重新上架賣的時間會被拖很長(?

https://tw.news.yahoo.com/news/%E7%9B%B8%E8%A6%AA%E9%9D%A0%E6%8A%95%E5%B9%A3-%E6%97%A5%E6%8E%A8-%E7%B5%90%E5%A9%9A%E5%B0%8D%E8%B1%A1%E8%B2%A9%E8%B3%A3%E6%A9%9F-30%E5%80%8B%E7%BD%90%E9%A0%AD%E5%85%A8%E5%94%AE%E7%A9%BA-113255381.html

**Comment by 710388** [comment]
蠻有趣的一個文章，這我倒是沒想到，這可以讓人比較能夠邁出第一步

**Comment by yubo0oo** [comment]
對於害優的人很有價值˙
但是可能吸引到很多怪人
